app/scripts/views/base.js

```diff
@@ -49,6 +50,14 @@ define(function (require, exports, module) {
       .slideDown(STATUS_MESSAGE_ANIMATION_MS)
       .attr('data-shown', 'true');
 
+    if (this.window.pageYOffset >= DEFAULT_HEADER_HEIGHT){
+      $success.css({
+        position: 'fixed',
+        top: 0,
+        width: 500 // required to force the width
```

Comment:
When testing on small screens, this causes it to be offset to the right. I tried `'100%'` and it seemed to work well?

Reply:
hmmm small screens should just do width 100% already anyway, let me check